tests/dialog-boxes.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { test, expect } from '@playwright/test';
+
+test.beforeEach( async({page}) => {
+  await page.goto('/')
+});
+
+test('Home page is opened and Welcome message is displayed', async ({page}) => {
+    /* 1. Select the PET TYPES menu item in the navigation bar */
+    // Navigate to the Pet Types page
+    await page.getByText('Pet Types').click()
+
+    /* 2. On the "Pet Types" page, add assertion of the "Pet Types" text displayed above the table with the list of pet types */
+    // Validate that h2 says "Pet Types" as expected
+    await expect(page.locator('h2')).toHaveText('Pet Types')
+  
+    /* 3. Click on "Add" button */
+    // Click the 'Add' button to add a new pet type, 'pig'
+    // There are a few elements with 'Add', so we specify 'button'
+    await page.getByRole('button', {name: 'Add'}).click()
+
+    /* 4. Add assertions of "New Pet Type" section title, "Name" header for the input field and the input field is visible */
+    // Validate that the new page section has "New Pet Type" as the header
+    // Expect that the new section header is below "Pet Types"
+    await expect(page.locator('h2').nth(1)).toHaveText('New Pet Type')
+
+    // Validate that the label for the input field is "Name"
+    await expect(page.locator('label')).toHaveText('Name')
+    
+    // Validate that the input field is visible
+    const input = page.locator('#name')
+    await expect(input).toBeVisible()
+
+    /* 5. Add a new pet type with the name "pig" and click "Save" button */
+    // Adding 'pig' as the new pet type
+    await input.click()
+    await page.keyboard.type('pig')
```

Comment:
Keep it simple. Not necessary to simulate the keyboard keystrokes. It was not a requirement.
Lines 35 and 36 can be just a single line: `await input.fill('pig')`

Reply:
Thanks! I will correct this and then merge.